packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTableRow/MyAdsTableRow.tsx

```diff
@@ -145,24 +145,24 @@ const MyAdsTableRow = ({ setIsModalOpen, ...rest }: TMyAdsTableProps) => {
                         Limits
                     </Text>
                     <Text color='less-prominent' size='sm'>
-                        {`Rate (1 ${account_currency})`}
+                        {`Rate (1 ${accountCurrency})`}
                     </Text>
                 </div>
                 <div className='p2p-v2-my-ads-table-row__line-details'>
                     <Text color={adPauseColor} size='sm'>
-                        {min_order_amount_display} - {max_order_amount_display} {account_currency}
+                        {minOrderAmountDisplay} - {maxOrderAmountDisplay} {accountCurrency}
                     </Text>
                     <Text color='success' weight='bold'>
                         <div className='display-layout'>
-                            {displayEffectiveRate} {local_currency}
-                            {rate_type === RATE_TYPE.FLOAT && (
-                                <AdType adPauseColor={adPauseColor} floatRate={rate_display} />
+                            {displayEffectiveRate} {localCurrency}
+                            {rateType === RATE_TYPE.FLOAT && (
```

Comment:
can we make a const for this since we also use it below

```suggestion
                            {isFloating && (
```